OrchardCore.Commerce/Views/PricePartSettings.Edit.cshtml

```diff
@@ -24,22 +24,20 @@
     </fieldset>
 </fieldset>
 
-<script at="Foot">
-$(function () {
-    $(document).ready(function () {
-        SpecificCurrencyGroupVisibility($('#@Html.IdFor(m => m.CurrencySelectionMode)'), $('#@Html.IdFor(m => m.SpecificCurrencyIsoCode)-Group'));
-    });
+<script at="Foot" depends-on="jQuery">
+    jQuery(function ($) {
+        $(document).ready(function () {
+            SpecificCurrencyGroupVisibility(
+                $('#@Html.IdFor(model => model.CurrencySelectionMode)'),
+                $('#@Html.IdFor(model => model.SpecificCurrencyIsoCode)-Group'));
+        });
 
-    $('#@Html.IdFor(m => m.CurrencySelectionMode)').change(function () {
-        SpecificCurrencyGroupVisibility($(this), $('#@Html.IdFor(m => m.SpecificCurrencyIsoCode)-Group'));
-    });
+        $('#@Html.IdFor(model => model.CurrencySelectionMode)').change(function () {
+            SpecificCurrencyGroupVisibility($(this), $('#@Html.IdFor(model => model.SpecificCurrencyIsoCode)-Group'));
```

Comment:
I'm not in the mood for a debate on principles right now, so let's keep it.

Reply:
Sorry to hear that. Get well soon. 😁

Reply:
Thanks 😀 Already doing much better.

I've also come to the conclusion that my being used to the short form got in the way of appreciating the more telling names; especially when you use `settings` instead of `model`, it actually makes a lot of sense. 👍🏼